lib/Alchemy/Phrasea/Core/Provider/StatusServiceProvider.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of Phraseanet
+ *
+ * (c) 2005-2014 Alchemy
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Alchemy\Phrasea\Core\Provider;
+
+use Alchemy\Phrasea\Status\CacheStatusStructureProvider;
+use Alchemy\Phrasea\Status\StatusStructureFactory;
+use Alchemy\Phrasea\Status\XmlStatusStructureProvider;
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+
+class StatusServiceProvider implements ServiceProviderInterface
+{
+    public function register(Application $app)
+    {
+        $app['status.provider'] = $app->share(function() use ($app) {
+            $provider = new XmlStatusStructureProvider($app['root.path'], $app['locales.available']);
```

Comment:
Having `XmlStatusStructureProvider` class with the "Provider" suffix is a bit confusing with Silex providers. Since it also perform writes on the XML too, maybe `XmlStructureAdapter` is suitable.